backend/pong/constants.py

```diff
@@ -1,17 +1,26 @@
-# constants.py
+#Movement units, and angles
 PLAYER_MOVEMENT_UNIT = 1
+BALL_SPEED = 5 
+MAX_BOUNCE_ANGLE = 90  # Bounce angle in degrees
+MIN_BOUNCE_ANGLE = -90
```

Comment:
isn't better to use radians?

Reply:
Agreed. I'm working on the game rn because of some bugs and have changed that already